starknet_devnet/blueprints/rpc/misc.py

```diff
@@ -63,15 +63,18 @@ async def syncing() -> Union[dict, bool]:
     return False
 
 
-# pylint: disable=too-many-arguments
-async def get_events(
-    from_block: BlockId,
-    to_block: BlockId,
-    chunk_size: int,
-    address: Address = None,
-    keys: List[Address] = None,
-    continuation_token: str = "0",
-) -> str:
+def try_parse_from_filter(filter_data, name):
```

Comment:
The pythonic way to do this is very simple:
```python
filter_data.get(key)
```

This defaults to returning `None` if the key is not present. In fact it can return any default value using the second argument, but the implicit default is `None`. Docs: [link](https://python-reference.readthedocs.io/en/latest/docs/dict/get.html)